### ISSUE TITLE
Allow user to specify population color

### ISSUE THREAD

**Issue by openjck** [mozilla/metrics-graphics#806]
This is offered by some other charting libraries, like [Chart.js](http://www.chartjs.org/).

It can be very helpful because it allows developers to use color generator like [distinct-colors](https://www.npmjs.com/package/distinct-colors) to generate colors for population lines that are maximally unique given the number of populations being shown. "If there are two lines, make them blue and red. If there are 60 lines, make them red, reddish-pink, reddish-orange, etc." This is obviously harder to do in CSS.

Alternatively, if metrics-graphics could do something like this natively, that would be even better.

**Comment by openjck** [comment]
Any way we can bump priority on this? It's not the biggest thing slowing Ensemble down, but it's one of the factors, as described here:

https://github.com/mozilla/ensemble/issues/91

**Comment by openjck** [comment]
Chart.js allows the developer to choose a line color using the `borderColor` and `backgroundColor` properties.

For example, this code:

```javascript
var lineChartData = {
    labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
    datasets: [{
        label: 'My First dataset',
        borderColor: window.chartColors.red,
        backgroundColor: window.chartColors.red,
        fill: false,
        data: [
            // ...
        ],
        yAxisID: 'y-axis-1',
    }, {
        label: 'My Second dataset',
        borderColor: window.chartColors.blue,
        backgroundColor: window.chartColors.blue,
        fill: false,
        data: [
            // ...
        ],
        yAxisID: 'y-axis-2'
    }]
};
```

... generates this chart:

<img width="1091" src="https://user-images.githubusercontent.com/933396/39221876-706a76de-4807-11e8-9be6-ef04bd323df7.png">

Of course it would have to be slightly different in metrics-graphics, given that metrics-graphics takes data in a different format. How about something like this:

```javascript
MG.data_graphic({
    data: data,
    target: '#examples',
    line_colors: ['#000000', '#ff0000, '#00000f'],
});
```

**Comment by hamilton** [comment]
I believe you're looking for the `color` argument, correct?

[This example](https://iodide-project.github.io/master/?gist=hamilton/ed4d74f9b0844ae2db49f2b9ca0c40a5) (click "run all cells" in the top left, little fast forward icon) seems to mirror the functionality in question.

**Comment by openjck** [comment]
This may do the trick. I'll look into it soon. I'm sorry I didn't know about this option. I didn't see it on the [List of options](https://github.com/mozilla/metrics-graphics/wiki/List-of-Options) page.

**Comment by hamilton** [comment]
Yep this is clearly a weakness in our bad / outdated documentation!

**Comment by hamilton** [comment]
@openjck updated the wiki to reflect this. To be honest, I did a massive feature push about a year and a half ago on MG, and I realized I never updated the documentation at all. A reckoning is upon us, feature-wise.

**Comment by openjck** [comment]
Great! Where was the change made? I don't see an update to *List of Options* but maybe I should be reading docs somewhere else?
